### PR TITLE
Installation

### PR DESCRIPTION
**Pull request by AminaBuhari**
Hi Reviewer 😀, 
In this task, my partner @oshanedesign and I @AminaBuhari  designed a web application from the data gotten from the SpaceX API. The application will allow users to book rockets and join selected space missions.
To highlight the task 🎯 We: - 

- Created a Kanban board and distributed the task which is available on the projects page. 

Happy review.

### CODE REVIEW

File: src/Components/NavigationBar.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  <div className="Nav">
+    <div className="logoDiv">
+      <img id="navLogo" src={Planet} alt="planetLogo" />
+      <h1 id="headerText">Space Traveler Hub</h1>
+    </div>
+    <ul id="navList">
+      <li><Link id="anchora" to="/Rockets">Rockets</Link></li>
+      <li><Link id="anchorb" to="/Missions">Missions</Link></li>
+      <li><Link id="anchorc" to="/">My Profile</Link></li>
+    </ul>
```
Comment (bimbolabuari) [inline src/Components/NavigationBar.js]: As required, Kindly use `<NavLink />` for the page navigation link
>image

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi Team , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- App works as required


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Components/NavigationBar.js (hunk 1 above)
Comment (ocrearyy) [reply src/Components/NavigationBar.js]: Hi @bimbolabuari , thanks for that. Changes added . 
e64

**Comment by omar-labana** [comment]
# **Approved ✔️**

Hi @AminaBuhari and @oshanedesign 

Congratulations on completing this project! 👏🏼 🎉

You have correctly implemented all of the requirements📃 correctly and without any errors⚠️ or bugs🪲.

Cheers and good luck in your next milestones! 🍀

Feel free to leave any questions or comments in the PR thread(don't forget to tag me 😄) if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**